lib/rust/ensogl/core/src/display/layout/alignment.rs

```diff
@@ -4,110 +4,177 @@ use crate::prelude::*;
 
 
 
-// =================
-// === Alignment ===
-// =================
+// ===============================
+// === 1-dimensional Alignment ===
+// ===============================
 
-/// Alignment abstraction. Example usage is camera origin placement allowing it to behave correctly
-/// when scaling the scene.
-#[derive(Clone, Copy, Debug)]
+/// Alignment. In one dimension the alignment is either start, center, or end. In 2 dimensions,
+/// [`Start`] means "left" horizontally and "bottom" vertically, while [`End`] means "right"
+/// horizontally and "top" vertically.
```

Comment:
Is the first sentence a title? If yes, it should be separated with two newlines like this:

```suggestion
/// Alignment. 
/// 
/// In one dimension the alignment is either start, center, or end. In 2 dimensions,
/// [`Start`] means "left" horizontally and "bottom" vertically, while [`End`] means "right"
/// horizontally and "top" vertically.
```